src/main/kotlin/de/docs_as_co/intellij/plugin/drawio/editor/DrawioWebView.kt

```diff
@@ -39,6 +40,14 @@ class DrawioWebView(lifetime: Lifetime) : BaseDrawioWebView(lifetime) {
         _xmlContent.set(null) // xmlLike is not xml
         send(OutgoingMessage.Event.Load(xmlLike, 1))
     }
+
+    fun loadPng(payload: ByteArray) {
+        _xmlContent.set(null) // xmlLike is not xml
+        val xmlLike = "data:image/png;base64,"+Base64.getEncoder().encodeToString(payload)
```

Comment:
We should probably setup a formatting linter ;)
```suggestion
        val xmlLike = "data:image/png;base64," + Base64.getEncoder().encodeToString(payload)
```